src/SDKs/DataBox/AzSdk.RP.props

```diff
@@ -0,0 +1,7 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!--This file and it's contents are updated at build time moving or editing might result in build failure. Take due deligence while editing this file-->
+  <PropertyGroup>
+    <AzureApiTag />
```

Comment:
@kushalvms we have a bug currently that will not update the meta data.
Can you build your project from command line once
msbuild build.proj /t:Build /p:Scope=SDKs\DataBox

Reply:
@kushalvms another thing you will have to do is get latest from upstream and generate using generate.ps1
This will create all the necessary meta data required for the PR to be merged in.

Reply:
locally the code generation is failing with latest autorest package version: 2.0.4262. I tried with the last version: 2.0.4215 and it passed. What am I missing here or is there any problem with the latest version?